proposals/PickAPI/README.md

```diff
@@ -0,0 +1,192 @@
+def Xform "VeryLargeForest" {
+    def PointInstancer "OakTrees" {
+        rel prototypes = <./OakTree_1>
+        def Xform "OakTree_1" {
+            def Mesh "Trunk" { ... }
+            def PointInstancer "Leaves" (
+                append apiSchemas = "PickAPI"
+            ) {
+                token pick:visibility = "inherited"
+                token pick:retargeting = "replace"
+                rel pick:targets = <.>
+
+                rel prototypes = <./Leaf_1>
+                def "Leaf_1" { ... }
+            }
+        }
+    }
+}
+```
+
+### Room (Retargeting, Inheritance, and Invisibility)
+This example demonstrates custom picking behavior for several scopes.
+- Use retargeting to suppress picking of prims
+- Overriding retargeting when nesting `PickAPI` application
+- Using guide geometry as a target for unpickable prims
+
+```
+# The room specifies that by default, prims are visible to picking (if imaged)
+# but that they should retarget to an empty list. By default, objects in the
+# room act as pick occluders.
+def Xform "Room" (
+    append apiSchemas = "PickAPI"
+) {
+    uniform token pick:visibility = "inherited"
+    uniform token pick:retargeting = "replace"
+    rel pick:targets = []
+
+    def Xform "Walls" {
+        def Mesh "NorthWall" { ... }
+        def Mesh "SouthWall" { ... }
+        def Mesh "EastWall" { ... }
+        def Mesh "WestWall" { ... }
+    }
+
+    # The door overrides the retargeting behavior so that it and all its
+    # descendants are pickable and retarget to the rotatable 'DoorGroup'
+    def Xform "DoorGroup" (
+        append apiSchemas = "PickAPI"
+    ) {
+        uniform token[] xformOpOrder = ["xformOp:rotateZ"]
+        double xformOp:rotateZ = 0.0
+
+        uniform token pick:visibility = "inherited"
+        uniform token pick:retargeting = "replace"
+        rel pick:targets = <.>
+
+        def Mesh "Door" { ... }
+        def Mesh "DoorKnob" { ... }
+        def Mesh "DoorKnocker" { ... }
+    }
+
+    # The window makes its glass not pickable by default but provides a
+    # tiny sphere collocated with Glass to be used as a pick target by
+    # users when guides are enabled.
+    def Xform "Window" {
+        def Mesh "Frame" { ... }
+        def Mesh "Glass" (
+            append apiSchemas = "PickAPI"
+        ) {
+            uniform token pick:visibility = "invisible"
+        }
+        def Sphere "GlassPickTarget" (
+            append apiSchemas = "PickAPI"
+        ) {
+            uniform token purpose = "guide"
+            uniform token pick:visibility = "inherited"
+            uniform token pick:retargeting = "replace"
+            rel pick:targets = <../Glass>
+        }
+    }
+}
+```
+
+## Questions
+* This proposal does not address support for `UsdGeomSubset`. Visibility and purpose are currently not supported by subsets, so it may be hard to support the visibility attribute. However, one can imagine a desire for using face sets to describe pick retargeting.
```

Comment:
Retargeting based on face-sets is a clear need for character models, for sure.  Subsets are the obvious answer that would fit elegantly with this proposal... modulo the issue you called out here.  Making Subset Imageable has been discussed and pretty much agreed to on usd-interest (the not-fully-resolved issue is that we have no way of rectifying a different `purpose` for a Subset from its parent Mesh, in current "face" usage).  More recently, though, we have been planning on migrating visibility and purpose out of Imageable and into VisibilityAPI, which would remove the "technical" impediment completely.